<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>application_window</class>
 <widget class="QMainWindow" name="application_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>application_window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add Block</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Renderer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Display</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Scene</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QComboBox" name="comboBox_3"/>
      </item>
      <item row="0" column="11">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Render</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Zoom</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="5">
       <widget class="QSlider" name="zoomSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="sliderPosition">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="8" colspan="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Fit Scene</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="10">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="12">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionShader_Properties"/>
    <addaction name="actionCode_Preview"/>
    <addaction name="actionExport_Scene"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionGroup"/>
    <addaction name="actionUngroup"/>
    <addaction name="actionEdit_Block"/>
    <addaction name="actionDelete_Blocks"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Grid"/>
    <addaction name="actionSnap_Blocks"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionShader_Properties">
   <property name="text">
    <string>Shader Properties</string>
   </property>
  </action>
  <action name="actionCode_Preview">
   <property name="text">
    <string>Code Preview</string>
   </property>
  </action>
  <action name="actionExport_Scene">
   <property name="text">
    <string>Export Scene</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="text">
    <string>Group</string>
   </property>
  </action>
  <action name="actionUngroup">
   <property name="text">
    <string>Ungroup</string>
   </property>
  </action>
  <action name="actionEdit_Block">
   <property name="text">
    <string>Edit Block</string>
   </property>
  </action>
  <action name="actionDelete_Blocks">
   <property name="text">
    <string>Delete Blocks</string>
   </property>
  </action>
  <action name="actionShow_Grid">
   <property name="text">
    <string>Show Grid</string>
   </property>
  </action>
  <action name="actionSnap_Blocks">
   <property name="text">
    <string>Snap Blocks</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
